src/main/kotlin/lotto/dto/LottoResult.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.dto
+
+import lotto.vo.LottoNumber
+
+data class LottoResult private constructor (private val _winningNumber: List<LottoNumber>) {
+    val winningNumber
+        get() = _winningNumber
+
+    companion object {
+        private const val WINNING_NUMBERS_DELIMITER = ","
+
+        fun of(winningNumbers: String?): LottoResult {
+            require(winningNumbers != null) { "당첨 번호는 null이 될 수 없습니다." }
```

Comment:
아래와 같이 변경해볼 수도 있을 것 같아요. 😉
```suggestion
            requireNotNull(winningNumbers) { "당첨 번호는 null이 될 수 없습니다." }
```

Reply:
정말 유용하네요! 👍